src/language/typescript/3.0/serializers/operation-object.ts

```diff
@@ -247,6 +259,32 @@ export const serializeOperationObject = combineReader(
 
 		const serializedResponses = serializeResponsesObject(from)(operation.responses);
 
+		const responseType: XHRResponseType = pipe(
+			lookup('200', operation.responses),
```

Comment:
What about 201 and other successful responses? AFAIK we have a list of them somewhere in a constant.

Reply:
fixed